internal/distro/rhel84/distro.go

```diff
@@ -1669,7 +1669,6 @@ func newDistro(name, modulePlatformID, ostreeRef string, isCentos bool) distro.D
 		uefi: false,
 	}
 	s390x.addImageTypes(
-		tarImgType,
```

Comment:
```
Building `tar` image for `s390x` on RHEL-84 ends with panic:
"s390x image must have a partition table, this is a programming error"
```

A tar image should not need a partition table, so this error message is funny. Removing it surely will fix the issue but I wonder why we are seeing this in the first place.